.circleci/config.yml

```diff
@@ -549,7 +548,7 @@ workflows:
       - simulate_sequence:
           name: simulate_sequence_eth
           network: "eth"
-          tasks: ""
+          tasks: "008-deputy-pause-module"
```

Comment:
I don't think this needs to be added to the sim sequence stuff as the new simulate stack functionality will pick it up because you're using the new system and we don't have any draft tasks in the old system. However, it doesn't do any harm.

Reply:
Oh interesting, I didn't know that the new system will pickup!
How can I know what I should use here to use the new system with my 2 PRs?